examples/addons/container-monitoring/README.md

```diff
@@ -0,0 +1,127 @@
+
+If you have any questions regarding the Container Monitoring add-on, please reach us out thru [this](mailto:askcoin@microsoft.com) email.
+
+```json
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "kubernetesConfig": {
+        "addons": [
+          {
+            "name": "container-monitoring",
+            "enabled": true,
+            "config": {
+              "workspaceGuid": "<Azure Log Analytics Workspace Guid in Base-64 encoded>",
+              "workspaceKey": "<Azure Log Analytics Workspace Key in Base-64 encoded>"
+            }
+          }
+        ]
+      }
+    },
+    "masterProfile": {
+      "count": 1,
+      "dnsPrefix": "",
+      "vmSize": "Standard_DS2_v2"
+    },
+    "agentPoolProfiles": [
+      {
+        "name": "agentpool",
+        "count": 3,
+        "vmSize": "Standard_DS2_v2",
+        "availabilityProfile": "VirtualMachineScaleSets"
+      }
+    ],
+    "linuxProfile": {
+      "adminUsername": "azureuser",
+      "ssh": {
+        "publicKeys": [
+          {
+            "keyData": ""
+          }
+        ]
+      }
+    },
+    "servicePrincipalProfile": {
+      "clientId": "",
+      "secret": ""
+    }
+  }
+}
+
+```
+
+You can validate that the add-on is running as expected with the following commands:
+
+You should see two sets of omsagent pods 1). One omsagent replicat set in one of the node 2). Many omsagent demonset pods as there are agent nodes .
+All these pods should be in 'Running' state after executing:
+
+```bash
+kubectl get pods -n kube-system
+```
+## UX
+
+For more details on how to use the product, see [Azure Monitor for containers](https://docs.microsoft.com/en-us/azure/azure-monitor/insights/container-insights-overview)
+
+![Image of Azure Monitor for containers](../../../docs/static/img/azure_monitor_aks_engine.png)
+
+
+## Supported Matrix
+
+ Refer to [azuremonitor-containers-aks-engine](https://github.com/Microsoft/OMS-docker/blob/aks-engine/README.md) for the supported matrix, troubleshooting and supportability etc.
+
+
+## Disable Monitoring 
+
+After you enable monitoring of your AKS Engine cluster, you can stop monitoring the cluster if you decide you no longer want to monitor it. 
+
+- If you have onboarded the monitoring using the HELM chart, then you can disable monitoring by uninstalling the chart. Refer Uninstalling the Chart section in [azuremonitor-containers](https://github.com/helm/charts/tree/master/incubator/azuremonitor-containers)
+
+- If you have onboarded using the Container Monitoring addon, then you can stop monitoring either 
+
+      - kubectl delete -f <kubernetesmasteraddons-omsagent-daemonset.yaml>
+
+        or
+
+      - deleting omsagent daemonset and replicaset using Kubectl tool
+
+## Contact
+
+If you have any questions or feedback regarding the container monitoring add-on, please reach us out thru [this](mailto:askcoin@microsoft.com) email.
```

Comment:
s/thru/through ?

Reply:
s/add-on/addon

Reply:
For Addon also, there is inconsistent usage (add-on, addon) across the docs.  I believe, at some point, everything else will be updated to use "addon" so I can update to addon in my doc updates?

Reply:
correct

Reply:
thanks. fixed.